src/foraging_gui/TransferToNWB.py

```diff
@@ -163,14 +164,25 @@ class obj:
         'water_day_total': water_day_total,
 
         # Weight
-        'base_weight': float(_get_field(obj, 'BaseWeight', reject_list=[''])),
-        'target_weight': float(_get_field(obj, 'TargetWeight', reject_list=[''])),
-        'target_weight_ratio': float(_get_field(obj, 'TargetRatio', reject_list=[''])),
-        'weight_after': float(_get_field(obj, 'WeightAfter', reject_list=[''])),
+        'base_weight': float(_get_field(obj, 'BaseWeight')),
+        'target_weight': float(_get_field(obj, 'TargetWeight')),
+        'target_weight_ratio': float(_get_field(obj, 'TargetRatio')),
+        'weight_after': float(_get_field(obj, 'WeightAfter')),
         
         # Performance
         'foraging_efficiency': _get_field(obj, 'B_for_eff_optimal'),
         'foraging_efficiency_with_actual_random_seed': _get_field(obj, 'B_for_eff_optimal_random_seed'),
+
+        # Optogenetics
+        'laser_1_calibration_power': float(_get_field(obj.Opto_dialog, 'laser_1_calibration_power')),
+        'laser_2_calibration_power': float(_get_field(obj.Opto_dialog, 'laser_2_calibration_power')),
+        'laser_1_target_areas': _get_field(obj.Opto_dialog, 'laser_1_target',default='None'),
+        'laser_2_target_areas': _get_field(obj.Opto_dialog, 'laser_2_target',default='None'),
```

Comment:
A tip to reduce backward compatibility issues is to be mindful when you call something like `obj.xxx` you always catch the case where we didn't have that field in the past.

Reply:
Fixed.